### ISSUE TITLE
[Bug]: ⚠Some Game related Files & Folders doesn't have Proper DOCTYPE, META Tags and alt Text to Image Elements

### ISSUE THREAD

**Issue by NarendraKoya999** [kunjgit/GameZone#2771]
### Give a brief about the bug ✍️

Description:
- In this Path , GameZone\Games. So many Games Folders are there.
- But, In Some Folders and It's related HTML Files Does not Maintain <DOCTYPE html> and <meta> Tags. This is Serious Thing.
- Also, alt attribute and It's Suitable text Does not Given in Some Folders and It's Files.

### What is the expected behavior? 🤔

Description:
- Zero Bug Free Errors after these Changes.

### Provide step by step information reproduce the bug 📄

Description:
- Go To GameZone\Games Folders Path>
- Randomly Checking Some Files.
- See Errors  related to above I am Raised.

### Select program in which you are contributing

GSSoC23

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

**Comment by kunjgit** [comment]
Hey @NarendraKoya999 ! 
 Thank you for raising an issue 💗 
 You can self assign the issue by commenting /assign in comment 😀 
 Make sure you follow [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING GUIDELINES](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) 🚀 
 Don’t Forget to ⭐ our [GameZone🎮](https://github.com/kunjgit/GameZone)
 Make sure you join our [Discord🕹️](https://discord.gg/fgwk4XZfxG)

**Comment by kunjgit** [comment]
Hey @NarendraKoya999 ! Thank you so much for your raising the issue💗 
 It’s all yours, you can come anytime again and make some contributions! 🚀 
 Alone, we can do little, but together we can do so much! 😇
